feconf.py

```diff
@@ -20,18 +20,24 @@
 import datetime
 import os
 
+
 # Whether to unconditionally log info messages.
 DEBUG = False
 
 # The platform for the storage backend. This is used in the model-switching
 # code in core/platform.
 PLATFORM = 'gae'
 
+# This should be string comparison, since all environment variables
+# are converted to string
+IS_MINIFIED = os.environ.get('MINIFICATION') == 'True'
+
 # Whether we should serve the development or production experience.
 if PLATFORM == 'gae':
     DEV_MODE = (
-        not os.environ.get('SERVER_SOFTWARE')
-        or os.environ['SERVER_SOFTWARE'].startswith('Development'))
+        (not os.environ.get('SERVER_SOFTWARE')
+         or os.environ['SERVER_SOFTWARE'].startswith('Development'))
+        and not IS_MINIFIED)
```

Comment:
mmh I think there is still the need of setting `DEV_MODE` to false because it is the one used in many places like selecting which javascript and css to include(either minified version in `static/generated/prod `or normal files from `static/generated/dev`). Setting it to false will pretty much imitated what happens in prod environment. Otherwise we may need rewrite other places where we use `DEV_MODE` to  also look for `IS_MINIFIED`